src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import java.time.LocalDateTime;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    @JoinColumn(name = "movie_id")
+    private Movie movie;
+    @ManyToOne
+    @JoinColumn(name = "ciname_hall_id")

```

Comment:
There seems to be a typo in the `@JoinColumn` name attribute. It should match the column name in your database, which is likely `cinema_hall_id`, not `ciname_hall_id`.